crates/ra_ide/src/ssr.rs

```diff
@@ -57,9 +60,10 @@ pub fn parse_search_replace(
     parse_only: bool,
     db: &RootDatabase,
     position: FilePosition,
+    selections: Vec<FileRange>,
```

Comment:
Aha, nice, this is quite general. Maybe rename `position` to `resolve_context` then?

Reply:
Done